index.js

```diff
@@ -2191,11 +2191,12 @@
     };
   }
 
-  //  see :: (String, { name :: String, url :: String }) -> String
+  //  see :: (String, { name :: String, url :: Nullable String }) -> String
```

Comment:
Let's use `String?` rather than `Nullable String` to reflect the fact that the field may not be present.